### PR TITLE
Add method for converting MeshElement from revit

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### Issues addressed by this PR

Adds method for converting any element to MeshElement from Revit

### CODE REVIEW

File: Revit_Core_Engine/Convert/Physical/FromRevit/MeshElement.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        [Output("beam", "BH.oM.Physical.Elements.Beam resulting from converting the input Revit FamilyInstance.")]
+        public static MeshElement MeshElementFromRevit(this Element element, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
+        {
+            settings = settings.DefaultIfNull();
+
+            MeshElement meshElement = refObjects.GetValue<MeshElement>(element.Id);
+            if (meshElement != null)
+                return meshElement;
+
+            Options meshOptions = Create.Options(ViewDetailLevel.Medium, true, true);
```
Comment (pawelbaran) [inline Revit_Core_Engine/Convert/Physical/FromRevit/MeshElement.cs:57]: Not entirely sure about including invisible geometries, but my gut feeling is that it should be `false`, assuming that physical namespace represents only the visible aspects of objects. On the other hand, I am 100% sure you do not need to compute references. So finally I would consider changing the code as follows:

```suggestion
            Options meshOptions = Create.Options(ViewDetailLevel.Medium, false, false);
```

**Review by pawelbaran — CHANGES_REQUESTED** [review]

**Review by pawelbaran — DISMISSED** [review]
Approving this PR without extensive testing on the basis of the code to be fully validated as a part of the wider workflow developed by @IsakNaslundBh

**Review by pawelbaran — DISMISSED** [review]
Approving the latest tweak that fixes diffing - however, could you please update the title @IsakNaslundBh?

**Review by pawelbaran — APPROVED** [review]
This PR looks good now I believe - happy to approve and continue the review on other threads.

File: Revit_Core_Engine/Convert/Physical/FromRevit/MeshElement.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
```
Comment (FraserGreenroyd) [inline Revit_Core_Engine/Convert/Physical/FromRevit/MeshElement.cs:3]:
```suggestion
 * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Review by pawelbaran — DISMISSED** [review]
Approving on the basis of the previous review.

**Comment by pawelbaran** [comment]
@BHoMBot check compliance
@BHoMBot check core

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Review by pawelbaran — DISMISSED** [review]
Approving on the basis of the previous review.

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Review by pawelbaran — APPROVED** [review]
Approving on the basis of previous reviews

**Comment by pawelbaran** [comment]
@BHoMBot check core
@BHoMBot check null-handling
@BHoMBot check serialisation
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am authorising dispensation to be granted on check ref. 10628311344

**Pull request merged**